retest_config.yaml

```diff
@@ -0,0 +1,143 @@
+receivers:
```

Comment:
Why do we need this `restest_config.yaml` file in source. I believe this was only for local testing purposes. Delete it.